torchvision/prototype/datasets/__init__.py

```diff
@@ -1,11 +1,10 @@
 try:
     import torchdata
-except (ModuleNotFoundError, TypeError) as error:
+except ModuleNotFoundError:
```

Comment:
We can drop the `TypeError` here since that was the error type of the legacy `torchdata` package hosted on PyPI. Since we took over the name, `pip install torchdata` will give you the correct package.